### DISCUSSION TITLE
Not able to get it to work on bungeecord

### DISCUSSION THREAD

**Discussion by Deeptanil** [games647/FastLogin#730] [General]
I've authme on my network.
And I've MySQL setup with it.
I've followed the instructions but not getting it to work... Just says sending request.

1. Do i set up MySQL as authme is using MySQL? And do i use the same database?
2. Do i have the same config on both bukkit and bungee?

Can I put the plugin ONLY in my bukkit server (hub), and run it normally without putting and bungee stat-id? Players will join, get logged in, and the MySQL connection will share the data with other servers.

Also y'all should make a discord server! If you need help i can help!

**Comment by TuxCoding** [comment]
Check the proxy server logs for any errors. 

>Can I put the plugin ONLY in my bukkit server (hub), and run it normally without putting and bungee stat-id?

No it's required for communication between the two plugins. The Spigot module performs the force login while the proxy module is responsible for managing the login verification process.

**Reply by Deeptanil** [reply]
> Check the proxy server logs for any errors.

There's no errors (that i can spot) on bungee.

**Reply by TuxCoding** [reply]
Please post the proxy log.

**Reply by Deeptanil** [reply]
Oh i see the issue, lmao it's because of the db

**Reply by Deeptanil** [reply]
Error
https://pastebin.com/6RtJMTZ9

**Reply by TuxCoding** [reply]
There you have it:

>.SQLNonTransientConnectionException: SSL Connection required, but not provided by server.

Either enable SSL in your MySQL server or disable it in the configuration file.

**Reply by Deeptanil** [reply]
Okay

**Reply by Deeptanil** [reply]
I'm guessing the config in bukkit is useless?

**Reply by TuxCoding** [reply]
Yes

**Reply by Deeptanil** [reply]
I've not put the plugin in my hub, instead I'm sending my user to the test server where i check if i get logged in.
https://mclo.gs/zeoSxcY look at the last attempt.

**Reply by TuxCoding** [reply]
The last attempt says the user already got logged in by AuthMe, so FastLogin cancelled it. So isn't this correct behavior?

**Reply by Deeptanil** [reply]
Uh, no because i was not logged in

**Reply by TuxCoding** [reply]
Then it's an issue with your auth plugin. As you can see that plugin already performed a login:

>[07:45:46] [Server thread/INFO]: [AuthMe] The user .Deeptanil has been automatically logged in, as requested via plugin messaging.

**Reply by TuxCoding** [reply]
Maybe because of the Java error message you got there.

>Could not save LimboPlayer for '.Deeptanil' [InaccessibleObjectException]: Unable to make private java.util.Collections$EmptyList() accessible: module java.base does not "opens java.util" to unnamed module @1fe20588

**Reply by Deeptanil** [reply]
Okay I'm gonna try something in a better bungee senerio tomorrow

**Reply by Deeptanil** [reply]
Got it to work!

**Reply by Deeptanil** [reply]
Thank you! 
Not sure how to close these!
